src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,25 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+import './TodoInfo.scss';
+
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo';
```

Comment:
[CHECKLIST ITEM 5] Prepare data in one place (App component) and pass it to child components. Don't use `import` keyword in non-root components to import data. Instead, pass the necessary data down as props from the parent component. In this case, the `UserInfo` component should receive the user data from its parent, rather than importing it directly.